Dockerfile

```diff
@@ -7,4 +7,5 @@ RUN apt update
 RUN apt install -y bash curl wget
 
 WORKDIR /
+COPY etcd_bootstrap_script.sh /var/etcd/bin/bootstrap.sh
```

Comment:
Do we need to make sure the file has the right permissions? Like [here](https://github.com/gardener/etcd-druid/blob/ba48ea629e59585d1b71060c9fd91c8778a60ad6/charts/etcd/templates/etcd-statefulset.yaml#L333). Or does the checked in file have the permissions already?

Reply:
Yes, the current file permissions on it is 755 (`rwxr-xr-x`), to allow execution from any user. So 356 permission will work. You can checkout the PR branch to confirm the file permissions.